.github/scripts/veristat-compare.py

```diff
@@ -0,0 +1,150 @@
+import sys
+import csv
+import os
+
+# File format:
+#
+# Columns:
+#  0. file_name
+#  1. prog_name
+#  2. verdict_base
+#  3. verdict_comp
+#  4. verdict_diff
+#  5. total_states_base
+#  6. total_states_comp
+#  7. total_states_diff
+#
+# Records sample:
+#  file-a,a,success,failure,MISMATCH,12,12,+0 (+0.00%)
+#  file-b,b,success,success,MATCH,67,67,+0 (+0.00%)
+
+TRESHOLD_PCT = 0
+
+HEADERS = ['file_name', 'prog_name', 'verdict_base', 'verdict_comp',
+           'verdict_diff', 'total_states_base', 'total_states_comp',
+           'total_states_diff']
+
+FILE        = 0
+PROG        = 1
+VERDICT_OLD = 2
+VERDICT_NEW = 3
+STATES_OLD  = 5
```

Comment:
is 4 missing on purpose?

Reply:
yes, sample of the record in the comment at the [start](https://github.com/kernel-patches/vmtest/pull/218/commits/88630c9ea660724f5017584bb260ea63645a7766#diff-d9bba8d01ee121eade3ef637cc83865fac60cb310a246ae82de3fd98c6dfd33eR5) of the file, here are the columns:

```
# Columns:
#  0. file_name
#  1. prog_name
#  2. verdict_base
#  3. verdict_comp
#  4. verdict_diff
#  5. total_states_base
#  6. total_states_comp
#  7. total_states_diff
```

The `verdict_diff` is ignored.